.github/schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Verified mods manifesto",
+    "description": "A list of mods that can be automatically downloaded by the Northstar client.",
+    "type": "object",
+    "patternProperties": {
+        "^[A-Za-z._0-9]+": {
+            "type": "object",
+            "properties": {
+                "DependencyPrefix": {
+                    "type": "string",
+                    "pattern": "^([A-Za-z._0-9]+)-([A-Za-z._0-9]+)$"
+                },
+                "Repository": {
+                    "type": "string",
+                    "pattern": "^https://"
```

Comment:
Should we even require GitHub repo or also allow other platforms? ^^

Reply:
I mean, I have nothing against GitLab, that's why I only forced the `https` prefix

Reply:
Works for me :D